### ISSUE TITLE
Recent Barkley Canyon MSc

### ISSUE THREAD

**Issue by kurtisanstey** [kurtisanstey/project#16]
@jklymak 

Steve mentioned that a student may have been using an instrument in Barkley Canyon for their MSc, when we first met with him. I had checked around, and found nothing, and it never came up in any of my Barkley Canyon research through Google Scholar or the UVic library. Until yesterday. 

https://scholarworks.sjsu.edu/cgi/viewcontent.cgi?article=8604&context=etd_theses

It looks as if they only examine 2013, but they do use both Axis and Upper Slope, as I do. However, and this could be a controversial personal opinion, it doesn't seem very 'polished'. What do you think? Is my project a bust, or do I need to shift focus? This is very worrying to me.

**Comment by kurtisanstey** [comment]
Use for reference!

**Comment by jklymak** [comment]
Can you maybe make a short synopsis and share it with your lazy advisor?  Also feel free to include any questions you may have - I'm happy to read it and try to answer.  Thanks!

**Comment by kurtisanstey** [comment]
Will do, Jody!

**Comment by kurtisanstey** [comment]
@jklymak 

This will be a great guide and comparison piece, but it doesn't seem very polished (to me, at least). Many sections are difficult to follow, which some terminology getting mixed up, and barely any information is given to describe how things were processed. I'm not even sure it was proof-read.

**Study Outline**
- The study seeks to investigate canyon effects on the local internal wave field, by comparing results in Barkley Canyon to previous studies in Monterey Canyon, La Jolla, etc. 
- Three ONC instrument sites are used to obtain 1-minute resolution velocity data: Rim (Upper Slope), Axis, and Slope (I'm unfamiliar with this site, but it's likely the 'Mid-East / Mid-West' ADCPs, as it's mentioned as a pair of 150 kHz instruments midway up the canyon wall). 
- There are two, two-month study periods, being spring (March/April 2013) and summer (June/July 2013).
- DFO Line P CTD data is used for climatology, with N determined as 0.0226 s-1 (this seems very high, and is possible a typo).
- The data was rotated by 30 degrees to match the canyon orientation with the continental slope.
- A generalised slope angle for the region was determined as 9.648 degrees.
- T_Tide MATLAB package used for analysis of the M2 tide.
- Principal axis determined using the princax function of the RPSstuff package by Rich Signell, to find the main flow direction at each depth bin for each ADCP. 
- pwelch used for PSD, with 8 segments, 50% overlap, and a Hanning window, for the depth bins at 16 MAB and 100 MAB.
- Wavelet analysis performed, but no spectrograms.
- A custom rotary process was made using Rick's book as a guide, and incorporating Welch averaging. This sounds similar to what I did, though they don't do a rotary or GM version. These are supplementary and are only in the Appendix.

**Seasonal Flow Patterns**
- For seasonal flow patterns a 33-hour half-amplitude filter was applied to remove tides and inertial effects. 
- At Axis in spring, stronger cross-slope velocities, and a persistent offshore/equatorward flow in the upper depths, than in summer.
- At Slope (Mid), quick switches from NW to SE flow, though there is strong horizontal banding in their data (I disregarded these mid-canyon ADCP because of this strong feature). 
- At Rim (Upper Slope), flow switching from the NW to the SE, with two-week spring periodicity, and monthly summer periodicity. 
- In general the flow is stronger as depth gets shallower, and in the summer.

**M2 Harmonic Analysis**
- Peak amplitude near the canyon rim (~-400 m), and high amplitudes closer to topography.
- Generally upward phase propagation indicates downward energy propagation, suggesting internal waves interacting with local topography. It's hard to follow, as I believe they mix up phase and energy propagation a few times.
- Higher amplitudes in summer.

**Principal Axis**
- Axis data shows evidence of flow being steered by the canyon topography that is notably missing, as expected, at the Rim (Upper Slope) location outside of the canyon. 
- There is little seasonal variance in the analysis.

**Rotated Spectra**
- Define a 'critical frequency' of 7.8e-5 Hz, which I assume has to do with the generalised slope angle.
- K1, f, and M2 dominate.
- Rim (Upper Slope) spring along-slope M2 is highest of all locations.
- Minimal seasonal changes for Axis and Slope (Mid).
- Summer PSD for Rim (Upper Slope) shows increase in K1, decrease in f near bottom.
- In general, more variance near the bottom suggests downward propagation of internal wave energy.
- Enhanced K1 in all cases, suggesting coastally trapped waves.

**Wavelet**
- Reinforces spectra findings.
- K1 stronger in summer, indicating coastally trapped waves.
- M2 also generally stronger in summer, and closer to the bottom.
- Subtle seasonal variations in periodicity.

**Discussion**
- Consistent K1 variance at all locations, sub-inertial at this latitude, suggest coastally trapped diurnal waves which studies suggest come from the mouth of the Juan de Fuca.
- Canyon ADCP should show elevated energy above the critical frequency, as V-shaped canyons amplify internal waves that enter from offshore, leading to high-frequency dispersive processes.
- Out of phase M2 energy for Axis vs Rim (Upper Slope) suggests that Axis gets mostly locally generated internal waves, while Rim gets locally and remotely generated waves.
- Upward M2 phase is indicative of downward internal wave energy in canyons.
- There is more energy near the bottom, particularly at Axis.
- Summertime reduces M2, and increases K1, and is assumed to be dependent on the regional currents/upwelling cycle.

**Comment by jklymak** [comment]
Great job Kurtis, lets discuss tomorrow, I appreciate your going through and doing this.

**Comment by kurtisanstey** [comment]
Use this MSc as a guide and comparison piece, and build on the seasonal research that was done. For example, may want to investigate phase propagation, but probably focus on broadband features and wind forcing in the inertial band.
